tabs/configuration.js

```diff
@@ -511,20 +511,20 @@ TABS.configuration.initialize = function (callback, scrollPosition) {
 
             function save_serial_config() {
                 if (semver.lt(CONFIG.apiVersion, "1.6.0")) {
-                    MSP.send_message(MSP_codes.MSP_SET_CF_SERIAL_CONFIG, MSP.crunch(MSP_codes.MSP_SET_CF_SERIAL_CONFIG), false, save_misc);
+                    MSP.send_message(MSP_codes.MSP_SET_CF_SERIAL_CONFIG, MSP_crunch(MSP_codes.MSP_SET_CF_SERIAL_CONFIG), false, save_misc);
```

Comment:
These call now look very unfortunate. If call to `MSP.send` requires a call to `MSP_crunch` these should really be defined as methods on the same object.